### ISSUE TITLE
Error: <rect> attribute width: Expected length, "NaN".

### ISSUE THREAD

**Issue by yaroslavert** [adobe-webplatform/Snap.svg#532]
my code 
let s = Snap(800,400);
let style = {
	fill: '#387',
	stroke: '#222',
	strokeWidth: 5
}
let pathAttr ={
	fill: 'transparent',
	stroke: '#222',
	strokeWidth: 3
}
let path = s.path("").attr(pathAttr);
let pathArr = [];
let updatePath = ()=>{
	let first = pathArr[0];
	let pathString = `M${first.x},${first.y}`;
	for(let i=1; i<pathArr.length; i++){
		let curent = pathArr[i];
		pathString+=`L${curent.x},${curent.y}`;
	}
	path.attr({
		d:pathString
	})
}
s.click(function(e) {
	if(e.target.tagName!="circle"){
		let circle = s.circle(e.offsetX,e.offsetY,15)
		 .attr(style)
		 .data('i',pathArr.length)
		 .mouseover(function(){
			this.stop()
				.animate({r:25},1000,mina.elastic)
		 })
		 .mouseout(()=>{
			this.stop()
				.animate({r:15},1000,mina.elastic)
		 })
		 .drag(function (dx, dy, x, y, e){
		 	this.attr({
		 		cx:x,
		 		cy:y
		 	});
		 	currentNode = pathArr[this.data('i')];
		 	currentNode.x = x;
		 	currentNode.y = y;
		 	updatePath();
		 })
		 

		pathArr.push({
			x:e.offsetX,
			y:e.offsetY
		})

		updatePath();
	}
});

**Comment by ibrierley** [comment]
Is it possible to put a testing example on a jsfiddle or jsbin ? It's hard to follow and examine from that.

**Comment by yaroslavert** [comment]
https://jsbin.com/huveruqipi/edit?js,output

**Comment by yaroslavert** [comment]
https://plnkr.co/edit/CSeFEJMEswmQtvDBH6Qx?p=preview

**Comment by ibrierley** [comment]
I think your problem is around...

.mouseout(()=>{ this.stop().animate({r:15},300,mina.elastic) })

And what is getting bound to 'this', as I think you're ending up animating the SVG paper. If you change it to a more typical function eg function() { this.stop....you will probably find it goes away.

**Comment by yaroslavert** [comment]
ok thank rapidly respond
